src/test/scala-2/io.circe.config/CirceConfigSpec.scala

```diff
@@ -105,7 +105,7 @@ class CirceConfigSpec extends AnyFlatSpec with Matchers {
 object CirceConfigSpec {
   val testResourcesDir = new java.io.File("src/test/resources")
   def resolveFile(name: String) = new java.io.File(testResourcesDir, name)
-  def readFile(path: String) = Source.fromFile(resolveFile(path)).getLines.mkString("\n")
+  def readFile(path: String) = Source.fromFile(resolveFile(path)).getLines().mkString("\n")
```

Comment:
Fixe the following warning:
```
[warn] /code/circe-config/src/test/scala/io.circe.config/CirceConfigSpec.scala:108:67: Auto-application to `()` is deprecated. Supply the empty argument list `()` explicitly to invoke method getLines,
[warn] or remove the empty argument list from its definition (Java-defined methods are exempt).
[warn] In Scala 3, an unapplied method like this will be eta-expanded into a function.
[warn]   def readFile(path: String) = Source.fromFile(resolveFile(path)).getLines.mkString("\n")
[warn]                                                                   ^
[warn] one warning found
```